cypress/integration/teacher-training-adviser.spec.js

```diff
@@ -2061,25 +2061,22 @@ describe("Feature - Get an adviser : Tests execution date and time : " + new Dat
 });
 
 describe("Hyperlink navigation check : Tests execution date and time : " + new Date(), () => {
-	it('Links through to "Scotland" page', function () {
+	it('Verify "Scotland" link', function () {
 		cy.goToUrl("");
-		cy.contains("a", "Scotland").invoke("removeAttr", "target").click();
-		cy.url().then((url) => {
-			expect(url).equal(Navlinks.teachInScotland);
+		cy.contains("a", "Scotland").should((link) => {
```

Comment:
Not a huge deal but these tests would run quicker if they were combined, so:

```
it("verify UK links", () => {
  cy.goToUrl("/");
  cy.contains("a", "Scotland")...
  cy.contains("a", "Wales")...
  cy.contains("a", "Nothern Ireland")...
});
```

At the minute we're needlessly reloading the page for each one

Reply:
Sure, I'll change it accordingly.  Thanks.